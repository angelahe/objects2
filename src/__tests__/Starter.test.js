import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Starter from "../components/Starter";
import renderer from "react-test-renderer";

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Starter />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
  shallow(<Starter />);
});

test('MathComp renders correctly', () => {
  const tree = renderer.create(<Starter />).toJSON();
  expect(tree).toMatchSnapshot();
});