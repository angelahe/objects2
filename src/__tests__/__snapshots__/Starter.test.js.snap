// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MathComp renders correctly 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <p>
    Edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
  <a
    className="App-link"
    href="https://reactjs.org"
    rel="noopener noreferrer"
    target="_blank"
  >
    Learn React
  </a>
</header>
`;
